tests/unit/acquisition/test_function.py

```diff
@@ -122,24 +122,21 @@ def test_expected_improvement(
     x_range = tf.cast(x_range, dtype=tf.float64)
     xs = tf.reshape(tf.stack(tf.meshgrid(x_range, x_range, indexing="ij"), axis=-1), (-1, 2))
 
-    class _Model(GaussianMarginal):
-        kernel = tfp.math.psd_kernels.MaternFiveHalves(variance_scale, length_scale=0.25).apply
+    kernel = tfp.math.psd_kernels.MaternFiveHalves(variance_scale, length_scale=0.25)
```

Comment:
stupid question but why is the lengthscale value hard coded but not the variance?

Reply:
cos length scale is to do with the shape of the function, or at least that's what I thought until we spoke about rbf and quadratic